### PR TITLE
Fix caller attribute set in TakeControl callback

### PR DESCRIPTION
**Pull request by tbimprob**

### CODE REVIEW

File: workers/csharp_physics/src/Main.cs [hunk 1]
```diff
@@ -57,11 +57,13 @@ private static int Main(string[] args)
...
             {
                 // Assign write ACL of 1001 to the worker ID received.
-                connection.SendLogMessage(LogLevel.Info, "Physics", "Assigning ClientData component to " + op.CallerAttributeSet[0]);
-                
+                connection.SendLogMessage(LogLevel.Info, "Physics", "Assigning ClientData component to " + op.CallerAttributeSet[1]);
+
                 Improbable.EntityAcl.Update aclUpdate = new EntityAcl.Update();
                 var currentWriteAcl = view.Entities[ourEntity].Get<EntityAcl>().Value.Get().Value.componentWriteAcl;
-                var workerAttributeSet = new WorkerAttributeSet(op.CallerAttributeSet);
+                Improbable.Collections.List<string> filteredCallerAttributeSet = new Improbable.Collections.List<string>();
```
Comment (dgavedissian) [inline workers/csharp_physics/src/Main.cs]: Nit: `var filteredCallerAttributeSet`.

File: workers/csharp_physics/src/Main.cs [hunk 2]
```diff
@@ -57,19 +57,21 @@ private static int Main(string[] args)
...
+                var workerAttributeSet = new WorkerAttributeSet(filteredCallerAttributeSet);
                 currentWriteAcl[ClientDataComponentId] = new WorkerRequirementSet(new List<WorkerAttributeSet>(new[] {workerAttributeSet}));
                 aclUpdate.componentWriteAcl.Set(currentWriteAcl);
                 connection.SendComponentUpdate(ourEntity, aclUpdate);
             });
 
             view.OnCommandResponse<ClientData.Commands.TestCommand>(op =>
             {
-                if (op.StatusCode == StatusCode.Success)
+                Iif (op.StatusCode == StatusCode.Success)
```
Comment (dgavedissian) [inline workers/csharp_physics/src/Main.cs]: Typo here

Comment (dgavedissian) [reply workers/csharp_physics/src/Main.cs]: @tbimprob poke

Comment (tbimprob) [reply workers/csharp_physics/src/Main.cs]: Woops. Fixed.

Comment (tbimprob) [reply workers/csharp_physics/src/Main.cs]: Just kicked off a CI run

**Review by dgavedissian — APPROVED** [review]

**Pull request merged**